misk/src/test/kotlin/misk/web/WebSocketsTest.kt

```diff
@@ -46,7 +46,7 @@ internal class WebSocketsTest {
     // Confirm interceptors were invoked.
     assertThat(logCollector.takeMessage(RequestLoggingInterceptor::class)).matches(
       "EchoWebSocket principal=unknown time=0.000 ns code=200 " +
-        "request=\\[JettyWebSocket\\[.* to /echo]] response=EchoListener"
+        "request=\\[JettyWebSocket\\[.* to /echo]] .* response=EchoListener .*"
```

Comment:
.* feels like it will make searches more difficult? Why output anything?

Reply:
What about requestHeaders=[] and likewise for response? The .* I think will be confusing for folks since it looks like a Regex and could be confused with being part of the logged request/response preceding it.

Reply:
Ah that is a matcher in the assertion. The logs already do something like `requestHeaders={accept-encoding=[gzip], connection=[keep-alive]}`